### PR TITLE
Dbet 876 bug dbets logs when claiming tokens

### PR DESCRIPTION
**Pull request by bradleySuira**

### CODE REVIEW

File: src/Casino/state/thunks.ts [hunk 1]
```diff
@@ -306,13 +311,21 @@ export function finalizeChannel(channelId, state) {
...
         dispatch,
-        getState,
+        _getState,
         { contractFactory }: IThunkDependencies
     ) => {
         try {
             const contract = await contractFactory.slotsChannelManagerContract()
-            const id = await contract.logChannelFinalized(channelId)
-            return await dispatch(actions.setChannelFinalized(id))
+            await contract
```
Comment (molekilla) [inline src/Casino/state/thunks.ts]: await eventemitter?

File: src/common/ContractFactory/contracts/BaseContract.js [hunk 2]
```diff
@@ -45,7 +45,8 @@ export default class BaseContract {
                     .pipe(
                         flatMap(() => from(promiseEvent)),
                         switchMap(i => of(i)),
-                        tap(() => {
+                        tap(i => {
+                            console.log('On tap', i)
```
Comment (molekilla) [inline src/common/ContractFactory/contracts/BaseContract.js]: remove console.log

Comment (bradleySuira) [reply src/common/ContractFactory/contracts/BaseContract.js]: done

File: src/Casino/state/thunks.ts (hunk 1 above)
Comment (bradleySuira) [reply src/Casino/state/thunks.ts]: yes, inside the method we call an async method `getPublicAddress` from the KeyHandler

**Pull request merged**